packages/babel-core/src/config/pattern-to-regex.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import path from "path";
-import escapeRegExp from "lodash/escapeRegExp";
```

Comment:
It might be possible to replace `lodash/escapeRegExp` with a function [provided by the MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Escaping), although I don't know whether it'd make sense to replicate that per-babel-package or whether it should go into a utility/common package if so.

Reply:
Could we instead consider replacing it with [escape-string-regexp](https://www.npmjs.com/package/escape-string-regexp).  That module is well tested, not being deprecated and gets 19 million weekly downloads (it's not below scrutiny).

Reply:
Keep in mind it requires major bump
https://github.com/sindresorhus/escape-string-regexp/blob/master/.travis.yml

Reply:
Ah sorry for missing that, I guess this suggestion would only apply to babel 8 then.